src/lsp/cobol_lsp/lsp_completion.ml

```diff
@@ -173,25 +126,25 @@ let expected_tokens base_env =
       | None -> snd (Srcloc.as_lexloc Srcloc.dummy)
       | Some Menhir.Element (_, _, _, pos) -> pos in
     if debug then (Lsp_io.log_debug "In State: %a" pp_env env; let tok = Expect.acceptable_terminals_in ~env in if List.length tok > 0 then Lsp_io.log_debug "Gained %d entries [%a]" (List.length tok) Fmt.(list ~sep:(any ";") Expect.pp_completion_entry) tok);
-    let productions = Expect.reducable_productions_in ~env in
-    let nullables = Expect.accaptable_nullable_nonterminals_in ~env in
+    let productions = Expect.reducible_productions_in ~env in
+    let nullables = Expect.acceptable_nullable_nonterminals_in ~env in
     let acc = Expect.CompEntrySet.add_seq
-      (List.to_seq @@ Expect.acceptable_terminals_in ~env) acc in
-    let acc = List.fold_left (fun acc (Menhir.X sym) ->
-      let default_value =
-        try Some (Cobol_parser.Recover.default_value sym)
-        with Not_found -> try Some (Expect.guessed_default_value_of_nullables sym)
-        with Not_found -> if debug then  Lsp_io.log_debug "Not found"; None in
-      Option.fold ~none:acc ~some:(fun a ->
-        let new_env = Menhir.feed sym pos a pos env in
-        if debug then Lsp_io.log_debug "NULLABLES: On %a\nGot to %a" pp_env_stack env pp_env_stack new_env;
-        inner new_env acc) default_value)
-    acc nullables in
+        (List.to_seq @@ Expect.acceptable_terminals_in ~env) acc in
+    let acc = List.fold_left (fun acc -> function
+        | Menhir.X T _ -> acc
+        | X N nt ->
+          let default_value = try Some (Expect.guessed_default_value_of_nullables nt)
```

Comment:
Here we need some newlines.

Reply:
The base for the sanitizer in `DEBUG.emit_nullable_unrecoverable`